LoopTests/ViewModels/BolusEntryViewModelTests.swift

```diff
@@ -409,7 +409,7 @@ class BolusEntryViewModelTests: XCTestCase {
         authenticateOverrideCompletion(.success(()))
         
         XCTAssertEqual(1.0, delegate.enactedBolusUnits)
-        XCTAssertEqual(false, delegate.enactedBolusAutomatic)
+        XCTAssertEqual(.manualRecommendationChanged, delegate.enactedBolusActivationType)
```

Comment:
Are there any tests covering the user accepting the recommendation?

Reply:
I'll check and add one if needed